etl/steps/data/grapher/migration/2024-11-20/migrant_stock_flows.py

```diff
@@ -0,0 +1,83 @@
+"""Load a garden dataset and create a grapher dataset.
+This grapher step has two purposes:
+1. Format the data in a way that is compatible with the grapher database (split into two tables and index on country and year).
+2. Add metadata programmatically to the data."""
+
+from etl.helpers import PathFinder, create_dataset
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+
+def run(dest_dir: str) -> None:
+    #
+    # Load inputs.
+    #
+    # Load garden dataset.
+    ds_garden = paths.load_dataset("migrant_stock")
+
+    # Read table from garden dataset.
+    tb = ds_garden.read("migrant_stock_dest_origin")
+
+    # grapher cannot handle this many indicators, so we remove male and female columns
+    tb = tb.drop(columns=["migrants_female", "migrants_male"])
```

Comment:
Ooh really! That's good to know. I wonder if it would work if they were in separate tables... but I don't think we need to try that out now for this!